NotesApi/NotesApi.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
     <PropertyGroup>
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
-        <NoWarn>$(NoWarn);1591</NoWarn>
+        <NoWarn>$(NoWarn);1591;CA1062;CA1303;CA1032;CA2227;CA1305</NoWarn>
```

Comment:
Maybe we can add this to the .editorconfig file as well

Reply:
I think they're already there, but this is for docker alone. I don't get any build issues/warnings in VS or running dotnet locally.